api/dimensions.go

```diff
@@ -55,13 +72,33 @@ func (api *DatasetAPI) getDimensions(w http.ResponseWriter, r *http.Request) {
 			return nil, err
 		}
 
+		totalCount := len(dimensions)
+
+		// sort.Slice(dimensions, func(i, j int) bool {
+		// 	return dimensions[i]["doc"]["name"].(string) > (dimensions[j]["doc"]["name"]).(string)
+		// })
+
+		// slicedDimensions := utils.Slice(dimensions, offset, limit)
+
```

Comment:
Do these commented out lines need to stay?

Reply:
Good spot. No they don't, must have missed those. Thanks!